hack/e2e.sh

```diff
@@ -230,6 +230,16 @@ function test_cassandracluster() {
         fail_test "First cassandra node not marked as seed"
     fi
 
+    if ! retry TIMEOUT=300 \
+         stdout_equals "${CASS_VERSION}" \
+         kubectl --namespace "${namespace}" \
+         get pilots \
+         --output 'jsonpath={.items[*].status.cassandra.version}'
```

Comment:
'*' here will select versions on all pilots, no? How does this work if there are 2 pilots returned? (or perhaps there's not 2 pilots at the point this test is run?)

Reply:
There's only one pilot at this point, but you're right it'd be better give index-0